### DISCUSSION TITLE
[5.x] User preference or config setting to hide sidebars by default

### DISCUSSION THREAD

**Discussion by FirePower-Capital** [craftcms/cms#15982] [Ideas] [answered]
Panel editing is becoming the norm in 5.x. I find myself constantly closing sidebars so I have enough room to view and edit content. Would love a user or general config setting to make "closed" the default state.

**Comment by brandonkelly** [comment, marked as answer]
Craft will keep track of whether the global sidebar was open or closed with a cookie, so a setting for that wouldn’t really make any sense; it would only ever be referenced when logging into the control panel from a new browser.

We’re discussing whether it makes sense to start doing the same for the details sidebar, too.

**Reply by brandonkelly** [reply]
We just released Craft 5.5.0, which now keeps track of whether detail sidebars and slideout sidebars should be open by default, across page loads. (#16025)
